<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="entradaPosicionX">
        <property name="placeholderText">
         <string>Ingresa X Inicial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="entradaPosicionY">
        <property name="placeholderText">
         <string>Ingresa Y Inicial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="entradaVelocidadX">
        <property name="placeholderText">
         <string>Ingresa Vel_X Inicial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="entradaVelocidadY">
        <property name="placeholderText">
         <string>Ingresa Vel_Y Inicial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="entradaMasa">
        <property name="placeholderText">
         <string>Ingresa la Masa del cuerpo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item row="1" column="0">
     <widget class="QGraphicsView" name="vistaSimulacion"/>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="botonCargarMasa">
        <property name="text">
         <string>CargarMasa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botonSimular">
        <property name="text">
         <string>Simular</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Detener">
        <property name="text">
         <string>Detener</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QListWidget" name="listaCuerpos"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
